src/miner.cpp

```diff
@@ -142,7 +142,7 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& s
       CBlockHeader h;
       nBlockSize += h.GetSerializeSize(SER_NETWORK, PROTOCOL_VERSION);
     }
-    assert(nBlockSize == 80);  // BU always 80 bytes
+    assert(nBlockSize == 80);  // BU block header is always 80 bytes
```

Comment:
absolutely, but is it appropriate for this PR?  half of you are telling me its too big, the other half make these additional changes :-)